pkgs/development/ruby-modules/bundled-common/default.nix

```diff
@@ -83,6 +84,8 @@ let
     ${maybeCopyAll mainGemName}
     cp ${gemFiles.gemfile} $out/Gemfile || ls -l $out/Gemfile
     cp ${gemFiles.lockfile} $out/Gemfile.lock || ls -l $out/Gemfile.lock
+
+    ${lib.concatMapStringsSep "\n" (path: "cp -r ${path} $out/") additionalConfPaths}
```

Comment:
can we use `lib.optionalString` for this bit?

Reply:
Wouldn't that still cause rebuilds because of the trailing newline if the array is empty?

Reply:
I see. yes, I wanted to avoid rebuilds. nvm then :).